tests/testsuite/global_cache_tracker.rs

```diff
@@ -778,10 +778,24 @@ fn clean_gc_dry_run() {
         .chain(cache.iter())
         .map(|p| p.to_str().unwrap())
         .join("\n");
+    let expected = {
+        let root = paths::root().display().to_string();
+        let re_crate = Regex::new(r"-[a-z0-9]{16}\b").unwrap();
+        let re_pack = Regex::new(r"pack-[a-z0-9]{40}").unwrap();
+        let p = expected_files.replace(&root, "[ROOT]");
+        let p = if cfg!(windows) {
+            p.replace("\\", "/")
+        } else {
+            p
+        };
+        let p = re_crate.replace_all(&p, "-[HASH]");
+        let p = re_pack.replace_all(&p, "pack-[HASH]");
+        format!("{p}\n")
+    };
```

Comment:
Let's skip this one

Reply:
done

Reply:
Note for similar case https://github.com/rust-lang/cargo/pull/14096#discussion_r1646855403